docs/source/guide/api_migrate.md

```diff
@@ -0,0 +1,180 @@
+ ---
+title: <img src="/images/LSE/en.svg" width=67 height=18 alt="Enterprise" style="vertical-align:middle"/> Update scripts and API calls
+type: guide
+order: 910
+meta_title: Update scripts and API calls to new version
+meta_description: Label Studio Enterprise documentation about updates and changes to the API endpoints in version 2.0. 
+---
+ 
+ With the new version of Label Studio Enterprise, you must update your scripts and API calls to match new API endpoints and formats. Some endpoints are new, some arguments for existing endpoints are deprecated and removed, and some payloads have changed for POST requests.
```

Comment:
Ah one important thing we need to add at the top - is that all object IDs are changed